### ISSUE TITLE
Ubuntu 16.04: libzmq5-dbg not installable/abandoned

### ISSUE THREAD

**Issue by imphil** [oerdnj/deb.sury.org#1365]
I'm on Ubuntu 16.04 amd64.
Enabled repo: `http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64`

I'm trying to install `libzmq5` and `libzmq5-dbg`. Both packages are in version 4.1.4 in Xenial (https://packages.ubuntu.com/search?suite=xenial&arch=any&searchon=names&keywords=libzmq5). 

With the PHP repository enabled, I get the following message from `apt-get`:

```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libzmq5-dbg : Depends: libzmq5 (= 4.2.2-1+ubuntu16.04.1+deb.sury.org+1) but 4.3.1-5+ubuntu16.04.1+deb.sury.org+1 is to be installed
E: Unable to correct problems, you have held broken packages.
```

Looking closer, I get:

```console
$ apt-cache policy libzmq5
libzmq5-dbg - lightweight messaging kernel (debugging symbols)
libzmq5:
  Installed: 4.3.1-5+ubuntu16.04.1+deb.sury.org+1
  Candidate: 4.3.1-5+ubuntu16.04.1+deb.sury.org+1
  Version table:
 *** 4.3.1-5+ubuntu16.04.1+deb.sury.org+1 500
        500 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 Packages
        100 /var/lib/dpkg/status
     4.1.4-7ubuntu0.1 500
        500 http://azure.archive.ubuntu.com/ubuntu xenial-updates/universe amd64 Packages
        500 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 Packages
     4.1.4-7 500
        500 http://azure.archive.ubuntu.com/ubuntu xenial/universe amd64 Packages

$ apt-cache policy libzmq5-dbg
libzmq5-dbg:
  Installed: (none)
  Candidate: 4.2.2-1+ubuntu16.04.1+deb.sury.org+1
  Version table:
     4.2.2-1+ubuntu16.04.1+deb.sury.org+1 500
        500 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 Packages
     4.1.4-7ubuntu0.1 500
        500 http://azure.archive.ubuntu.com/ubuntu xenial-updates/universe amd64 Packages
        500 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 Packages
     4.1.4-7 500
        500 http://azure.archive.ubuntu.com/ubuntu xenial/universe amd64 Packages
```

Looking at the repository at http://ppa.launchpad.net/ondrej/php/ubuntu/pool/main/z/zeromq3/, it seems that the **debug packages got renamed from `libzmq5-dbg` to `libzmq5-dbgsym`**.

Is this renaming intentional?

(Background story: I ran into this in GitHub actions, which by default enables your PHP repository in Ubuntu 16.04 runners. I have no need for PHP, but need ZeroMQ.)

**Comment by oerdnj** [comment]
Yes, dbgsym is the automatic debug symbol package.

**Comment by imphil** [comment]
Thanks for your answer. 

The problem is:
- the libzmq5 package from the PHP repository replaces the xenial-provided one. All good.
- the libzmq5-dbg package from the PHP repository is uninstallable, but still replaces the xenial-provided package.

Would it be possible to rename the -dbgsym package to -dbg to
1) Replace the uninstallable -dbg package in the PHP repository, and 
2) be consistent with the naming in xenial

**Comment by oerdnj** [comment]
No, not really. The dbgsym get created automatically.

**Comment by oerdnj** [comment]
Have you read https://github.com/oerdnj/deb.sury.org/wiki/Debugging-Symbols ?

**Comment by imphil** [comment]
I cannot really modify the list of packages that is being installed, there's another script in my setup that relies on `libzmq5` and `libzmq5-dbg` being available on Ubuntu 16.04.

Note that I'm not *intentionally* using the PHP repository, but it's activated by default in the Ubuntu 16.04 images provided by GitHub Actions. For now I'll just go with https://github.com/opensocdebug/osd-sw/blob/7c895a6e9bbcb35611db8981fec871de3ef270de/.github/workflows/ci.yml#L24-L28
